### PR TITLE
:ambulance: - Monorepo hotfix

### PR DESCRIPTION
**Pull request by jfrolich**

### CODE REVIEW

File: server/package.json [hunk 1]
```diff
@@ -1,38 +1,38 @@
...
-	"bugs": {
-		"url": "https://github.com/rescript-lang/rescript-vscode/issues"
-	},
-	"dependencies": {
-		"chokidar": "^3.5.1",
-		"vscode-jsonrpc": "^8.0.1",
-		"vscode-languageserver": "^8.0.1",
-		"vscode-languageserver-protocol": "^3.17.1"
-	}
+  "name": "@rescript/language-server",
```
Comment (jfrolich) [inline server/package.json:2]: for some reason this was tab formatted (and in the server it's spaces - I think spaces is more standard)

File: server/src/utils.ts [hunk 2]
```diff
@@ -70,7 +69,7 @@ export let findProjectRootOfFile = (
   } else {
     const isDir = path.extname(source) === "";
     return findProjectRootOfFileInDir(
-      isDir ? path.join(source, "dummy.res") : source
+      isDir && !allowDir ? path.join(source, "dummy.res") : source
```
Comment (jfrolich) [inline server/src/utils.ts:72]: This caused the fix to not work :|

**Comment by zth** [comment]
You need to run `npm install` in both folders as well so the version is updated in the lockfile.

**Comment by nojaf** [comment]
Hello @jfrolich, could you describe your folder structure a bit where this originally went wrong. What kind of setup/folder structure did you have where the root could not be found?
I'm looking at `findProjectRootOfFile` and would like to learn why this seems more complicated than I would expect it to be.

**Pull request merged**